msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionMessageBean.java

```diff
@@ -449,4 +457,128 @@ public String getAuthorEid(){
 		String userEid = this.getMessage().getCreatedBy();
  		return userEid;
 	}	
+
+	private boolean isAnonymousEnabled()
+	{
+		return ServerConfigurationService.getBoolean("msgcntr.forums.anonymous.enable", true);
```

Comment:
As Brian said above, probably something like AnonymousManager.SAK_PROP_ANON_ENABLE is better here